src/ci/docker/asmjs/Dockerfile

```diff
@@ -29,6 +29,11 @@ ENV EM_CONFIG=/emsdk-portable/.emscripten
 
 ENV TARGETS=asmjs-unknown-emscripten
 
-ENV RUST_CONFIGURE_ARGS --enable-emscripten
+ENV RUST_CONFIGURE_ARGS --enable-emscripten --disable-optimize-tests
 
-ENV SCRIPT python2.7 ../x.py test --target $TARGETS
+ENV SCRIPT python2.7 ../x.py test --target $TARGETS \
+  src/test/run-pass \
+  src/test/run-fail \
```

Comment:
I think we should still enable `run-pass-fulldeps`, `run-fail-fulldeps` and `run-make`.

Reply:
Oh currently I don't believe we run any of those tests, right?

Reply:
Oh let me check...